GoogleTestAdapter/Core/Runners/CommandLineGenerator.cs

```diff
@@ -210,20 +230,27 @@ private bool AllTestCasesOfExecutableAreRun()
 
         private List<TestCase> GetTestCasesNotRunBySuite(List<string> suitesRunningAllTests)
         {
-            var testCasesNotRunBySuite = new List<TestCase>();
-            // ReSharper disable once LoopCanBeConvertedToQuery
-            foreach (TestCase testCase in _testCasesToRun)
-            {
-                bool isRunBySuite = suitesRunningAllTests.Any(s => s == GetTestsuiteName(testCase));
-                if (!isRunBySuite)
-                {
-                    testCasesNotRunBySuite.Add(testCase);
-                }
-            }
-            return testCasesNotRunBySuite;
+            return _testCasesToRun.Where(t => !suitesRunningAllTests.Contains(GetTestsuiteName(t))).ToList();
```

Comment:
That's nice! Feel free to remove the other code...

Reply:
OK :)